packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -207,3 +207,48 @@
 .usa-modal--lg {
   max-width: rem-override(55rem);
 }
+
+.usa-alert {
+  border-left-width: rem-override(0.5rem);
+  .usa-alert__body {
+    font-size: rem-override(1.06rem);
+    max-width: rem-override(64rem);
+    padding: rem-override(1rem);
+  }
+}
+
+.usa-alert--info .usa-alert__body,
+.usa-alert--warning .usa-alert__body, 
+.usa-alert--success .usa-alert__body,
+.usa-alert--error .usa-alert__body {
```

Comment:
Just as an FYI, this could also be written as
```
.usa-alert--info, .usa-alert--warning, .usa-alert--success, .usa-alert--error {
  .usa-alert__body {}
}
```

Reply:
Cool! I'll give that try.